openprescribing/frontend/management/commands/create_views.py

```diff
@@ -88,17 +90,39 @@ def fill_views(self):
         pool.join()  # wait for all worker processes to exit
 
         for table in tables:
-            self.download_and_import(table)
+            self.download_and_import(table, prescribing_date)
             self.log("-------------")
 
-    def download_and_import(self, table):
+    def download_and_import(self, table, prescribing_date):
         storage_prefix = 'hscic/views/{}-'.format(table.name)
         exporter = TableExporter(table, storage_prefix)
 
-        with tempfile.NamedTemporaryFile(mode='r+') as f:
+        filename_base = '{}-{}'.format(table.name, prescribing_date)
+        raw_path = os.path.join(
+            settings.LARGE_TMP_DIR,
+            '{}-raw.csv'.format(filename_base)
+        )
+        sorted_path = os.path.join(
+            settings.LARGE_TMP_DIR,
+            '{}-sorted.csv'.format(filename_base)
+        )
+
+        self.log('Downloading {} to {}'.format(table.name, raw_path))
+        with open(raw_path, 'w') as f:
             exporter.download_from_storage_and_unzip(f)
-            f.seek(0)
 
+        self.log('Sorting {} to {}'.format(table.name, sorted_path))
```

Comment:
We should add a comment here explaining why we sort and why we do it this way. e.g. with a link to https://github.com/ebmdatalab/openprescribing/issues/698 (or just a summary of it).